TypeScript compiler can't found declaration

- `node` version - v8.6.0
- `celebrate` version - 5.0.1
- `joi` version (via `npm ls --depth=0 | grep joi`) -

*Explain the problem here*

Now TypeScript declaration https://github.com/continuationlabs/celebrate/blob/master/lib/index.d.ts is presented in npm package, but can't be found by TypeScript compiler:

```
$ ls node_modules/celebrate/lib/index.d.ts 
node_modules/celebrate/lib/index.d.ts
$ ./node_modules/.bin/tsc 

t.ts(1,28): error TS7016: Could not find a declaration file for module 'celebrate'. '/home/kostya/tmp/ts-test/node_modules/celebrate/lib/index.js' implicitly has an 'any' type.
  Try `npm install @types/celebrate` if it exists or add a new declaration (.d.ts) file containing `declare module 'celebrate';`

1 import * as celebrate from 'celebrate';
                             ~~~~~~~~~~~
```

For fix this need to add property `types` to `package.json` of `Celebrate` package:

```diff
{
+ "types": "./lib/index.d.ts"
}
```

--- a/package.json
+++ b/package.json
@@ -3,6 +3,7 @@
   "version": "5.0.1",
   "description": "A joi validation middleware for Express.",
   "main": "lib/index.js",
+  "types": "lib/index.d.ts",
   "scripts": {
     "lint": "belly-button -f",
     "test": "npm run lint && lab -v -a code -p -t 100"

